react/game/src/scss/variables/_colors.scss

```diff
@@ -1 +1,6 @@
 $square-border-color: #999;
+$my-blue: #4DD2FF;
```

Comment:
Try to be a little more descriptive on where the color is being used. Something like `x-blue` or `border-black`